main.js

```diff
@@ -0,0 +1,4 @@
+const ONE = 1;
+ONE = 2;
+var password;
+password=******;
```

Comment:
Expected an identifier and instead saw '*'.<br>Expected an operator and instead saw '*'.